### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by bhaggs** [G-Two/homeassistant-subaru#9] [General]
Just wanted to drop a note, that I'm really appreciative of the work you did. Fully expected nothing to exist to integrate sensor data from my Crosstrek into Home Assistant and was pleasantly surprised to find this library which is really well done and was quite easy to integrate. Thanks again.

**Comment by G-Two** [comment]
I'm glad it's working and that you find it useful. Thanks for the note!
